mw/language.d.ts

```diff
@@ -224,7 +218,7 @@ declare global {
              * Creates the data mw.Map if there isn't one for the specified language already.
              *
              * @param {string} langCode
-             * @param {string|Object} dataKey Key or object of key/values
+             * @param {string|Object.<string, Mixed>} dataKey Key or object of key/values
```

Comment:
In VS Code, there will be an error saying that the type `Mixed` cannot be found. Replacing it with `{string|Object.<string, any>}` can fix it.